whois-api/src/test/java/net/ripe/db/whois/api/elasticsearch/WhoisRdapElasticServiceTestIntegration.java

```diff
@@ -182,6 +183,7 @@ public void setup() throws IOException {
                 "admin-c:       PP1-TEST\n" +
                 "e-mail:        org@test.com\n" +
                 "mnt-by:        OWNER-MNT\n" +
+                "mnt-ref:       OWNER-MNT\n" +
                 "created:         2022-08-14T11:48:28Z\n" +
```

Comment:
can we add other attributes too in this for testing mnt_lower, mnt_domains etc?? so we can have one object to test all attributes?

Reply:
Done